src/utils/vector.cuh

```diff
@@ -370,6 +370,12 @@ VECATTR  float4 floorf(const float4 &a){
 
 VECATTR float dot(float4 a, float4 b){return a.x*b.x + a.y*b.y + a.z*b.z + a.w*b.w;}
 
+VECATTR float4 normalize(float4 v)
+{
```

Comment:
Use the same style as the rest of the file, "VECATTR float4 normalize(float4 v){"